### ISSUE TITLE
Trust HTTPS certificate on SLES

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/AspNetCore.Docs#28292]
Add instructions to [trust HTTPS certificate on RHEL](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-6.0&tabs=visual-studio#trust-https-certificate-on-linux)

SLES version of:

- [ ]  [Ubuntu trust the certificate for service-to-service communication](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-6.0&tabs=visual-studio#ubuntu-trust-the-certificate-for-service-to-service-communicatio)
- [ ] Trust HTTPS certificate on Linux using Edge or Chrome
- [ ] Trust the certificate with Firefox on Linux

Partial instructions [here](https://github.com/dotnet/aspnetcore/issues/32842)

related #28278
parent: #27603

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c7ddfb2f-5dea-5e31-2a27-02a4dc16104c
* Version Independent ID: 12ca0b9a-2638-a4ce-c45f-d4579c83dafb
* Content: [Enforce HTTPS in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-6.0&tabs=visual-studio#ubuntu-trust-the-certificate-for-service-to-service-communication)
* Content Source: [aspnetcore/security/enforcing-ssl.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/enforcing-ssl.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by djgalvan** [comment]
See #[32842](https://github.com/dotnet/aspnetcore/issues/32842#issuecomment-1418161969)

**Comment by Rick-Anderson** [comment]
@djgalvan  would you be able to help on this? 

The [SLES instructions](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/enforcing-ssl.md?plain=1#L447-L532) are commented out as they are a copy of the RHEL instructions. I have a SLES anyone testing this can use.

cc @selvinfehric if you're interested.

**Comment by djgalvan** [comment]
@Rick-Anderson I can spend some time on it over the week. I spent some time on it and recorded coverage in the parent issue. SLES documentation will involve adjusting the package manager from `dnf` to `zypper` and the package-dependency naming. @selvinfehric It's up for grabs if you'd like, I've got safety meetings to attend through the week so I'd be helping after hours.

**Comment by Rick-Anderson** [comment]
@djgalvan no hurry and any help appreciated.

**Comment by djgalvan** [comment]
Rick, I had submitted some changes using the web interface the morning after, I'm not seeing it anywhere. I'll check it out and try again some time this week.

**Comment by Rick-Anderson** [comment]
@djgalvan no PR was created. Hopefully you still have the changes.

**Comment by djgalvan** [comment]
No, I did it in the browser. I must be missing a second prompt/confirmation, second time it's gotten me lol. It didn't take long, I'll go over it again.

**Comment by Rick-Anderson** [comment]
Just change one character and do a PR, then it's saved and you can edit the file in the browser.
